app/javascript/listings/elements/categories.jsx

```diff
@@ -3,62 +3,66 @@ import { h, Component } from 'preact';
 
 class Categories extends Component {
   options = () => {
-    const { categoriesForSelect, category } = this.props
+    const { categoriesForSelect, category } = this.props;
     return categoriesForSelect.map(array => {
       // array example: ["Education/Courses (1 Credit)", "education"]
-      if(category === array[1]) {
-        return(
-          <option value={array[1]} selected>{array[0]}</option>
-        )
+      if (category === array[1]) {
+        return (
+          <option value={array[1]} selected>
+            {array[0]}
+          </option>
+        );
       }
-      return(
-        <option value={array[1]}>{array[0]}</option>
-      )
-    })
-  }
+      return <option value={array[1]}>{array[0]}</option>;
+    });
+  };
 
   details = () => {
-    const { categoriesForDetails } = this.props
+    const { categoriesForDetails } = this.props;
     const rules = categoriesForDetails.map(category => {
-      const paragraphText = `${category.name}: ${category.rules}`
-      return(
-        <p>
-          {paragraphText}
-        </p>
-      )
-    })
+      const paragraphText = `${category.name}: ${category.rules}`;
+      return <p>{paragraphText}</p>;
+    });
 
-    return(
+    return (
       <details>
-        <summary>
-          Category details/rules
-        </summary>
+        <summary>Category details/rules</summary>
         {rules}
       </details>
-    )
-  }
+    );
+  };
 
   render() {
-    const { onChange } = this.props
-    return(
+    const { onChange } = this.props;
+    return (
       <div className="field">
         <label className="listingform__label" htmlFor="category">
```

Comment:
Sorry if I'm missing something here but it says it's a label for an element with id "category" (htmlFor="category") but there's no element with id category.

Reply:
I will assume it is the <select> after that. Interesting that Eslint did not pick that up